core/templates/services/context.service.spec.ts

```diff
@@ -244,14 +319,59 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('skill_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+  });
+
+  describe('behavior in different pages', () => {
+    beforeEach(() => {
+      ecs = TestBed.get(ContextService);
+      urlService = TestBed.get(UrlService);
+    });
+
+    it('should correctly retrieve the page context', () => {
```

Comment:
Here and below. Duplicate description. Please make them more specific to what exactly is being tested.

Reply:
Done!